includes/Core/Assets/Assets.php

```diff
@@ -790,6 +798,34 @@ private function get_inline_dashboard_sharing_data() {
 		return apply_filters( 'googlesitekit_dashboard_sharing_data', $inline_data );
 	}
 
+	/**
+	 * Gets data relevant for `trackEvent` calls.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array The tracking inline data to be output.
+	 */
+	private function get_inline_tracking_data() {
+		$site_url     = $this->context->get_reference_site_url();
+		$current_user = wp_get_current_user();
+
+		$inline_data = array(
+			'referenceSiteURL' => esc_url_raw( trailingslashit( $site_url ) ),
+			'userIDHash'       => md5( $site_url . $current_user->ID ),
+		);
+
+		/**
+		 * Filters the most basic inline data to pass to JS.
+		 *
+		 * This should not include anything remotely expensive to compute.
+		 *
+		 * @since 1.2.0
```

Comment:
Could you change it to `n.e.x.t`?
```suggestion
		 * @since n.e.x.t
```